### DISCUSSION TITLE
Reset value in text fields

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#1646] [Proposals]
### Goal

Have the possibility to remove value in inputs fields in a easy way

### Context

Only the search field has this possibility
https://rebrand.ly/90nvxre

### Impact

Consistency between text field usage and improvements to the user to remove value in a easy way

### Discussion Questions

Should the rest of the elements have the same possibility?

**Comment by aweell** [comment]
Agree with this, I would have a prop like for example `isClearable` that allows to decide if you want to provide the clear functionality or not to the field.

Why this should be optional? If the user knows the value to enter and it’s less likely that he will change it later. In search fields is very useful since maybe I don't know the exact query or I want to make multiple queries.

For example in a form we're asking for personal data the chances a user need to reset a text field are very low.

> Should the rest of the elements have the same possibility? 

Following the previous statement, the chances of entering a wrong IBAN or CVV I think are also low.

The password case is also particular since we are already using the clear space for the hide/show feature
